server/models/UserModel.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
```

Comment:
wait this model already exists though right?

Reply:
Yes, I merged Umer's branch to this branch for code integrality. They are the same model, no change was made on it.